bibs/Rheda_Wiedenbrueck.json

```diff
@@ -20,6 +20,21 @@
             "returndate": -1,
             "status": 3
         },
+        "mediatypes": {
+            "0": "ALL",
+            "1": "BOOK",
+            "11": "DVD",
+            "12": "ART",
+            "2": "MAGAZINE",
+            "20": "AUDIOBOOK",
+            "25": "GAME_CONSOLE_WII",
+            "26": "GAME_CONSOLE_PLAYSTATION",
+            "3": "CD_SOFTWARE",
+            "30": "EBOOK",
+            "6": "CD_MUSIC",
+            "8": "SCORE_MUSIC",
+            "9": "BOARDGAME"
+        },
```

Comment:
mediatypes look wrong (see longer comment above), at least `ALL` needs to be removed.

Reply:
See my [comment](https://github.com/opacapp/opacapp-config-files/pull/11#discussion_r430272819) at Essen.

Reply:
see reply above.